src/views/v2/commerce/checkout.php

```diff
@@ -42,23 +42,23 @@ class="tribe-tickets__commerce-checkout"
 			<?php $this->template( 'checkout/cart', [ 'section' => $section ] ); ?>
 		<?php endforeach; ?>
 		<?php tribe( 'tickets.editor.template' )->template( 'v2/components/loader/loader' ); ?>
+		<?php $this->template( 'checkout/cart/empty' ); ?>
```

Comment:
We need to add the `@since TBD` and update the `@version TBD` for templates whenever we make changes.